src/Blorc.OpenIdConnect/Models/OidcProviderOptions.cs

```diff
@@ -17,6 +17,9 @@ public OidcProviderOptions()
         [JsonPropertyName("automaticSilentRenew")]
         public bool AutomaticSilentRenew { get; set; }
 
+        [JsonPropertyName("timeForAutomaticLogoutOnUserInactivity")]
+        public int TimeForAutomaticLogoutOnUserInactivity { get; set; }
```

Comment:
default should be 20 minutes. Or, even better, should be the session timeout from the server if we can fetch that.

Reply:
By default is zero and do nothing